build.assets/Dockerfile

```diff
@@ -1,24 +1,16 @@
-FROM quay.io/gravitational/debian-venti:go1.12.9-stretch
+FROM quay.io/gravitational/debian-venti:go1.13.8-stretch
```

Comment:
Why is 13.8 needed in the buildbox for this upgrade?

Reply:
Because of go modules, resolving dependencies in 1.13 works much better and with 1.12 as far as I remember I had huge problems with it.